web/src/components/Table/Table.tsx

```diff
@@ -240,46 +266,24 @@ export default function Table({ data, currentPage, rowCountPerPage, visibleColum
 
     if (Object.keys(filters).length > 0) {
       processedData = processedData.filter(row => {
-        return Object.keys(filters).every(column => {
-
-          
-        const columnIndex = config.columns.indexOf(column);
-        const cellValue = row[columnIndex]?.toString() || '';
-
+        return Object.keys(filters).every(column => { 
+          const columnIndex = config.columns.indexOf(column);
+          const cellValue = row[columnIndex]?.toString() || '';
 
           if (config.displayed_numeric_columns.includes(column)) {
-
-            const [minValue, maxValue] = filters[column].split(',').map(Number);
-
-            if (!minValue || !maxValue) return data;
-
-
-            const numericCellValue = parseFloat(cellValue);
-
-            return numericCellValue >= minValue && numericCellValue <= maxValue;
-
+            return filterByRange(cellValue, filters[column]);
           } else if (config.unique_column_values[column]) {
```

Comment:
I cannot understand why there is this if in here. Notice that you have two branches in the nested `if` statements that lead to the `filterByString` method. This can be even clearer. 

Moving the filtering into functions looks great though! Much better regarding readability and understandability! Nice work!

Reply:
It is to add filtering to the columns that are additional such as birth season but only 
```
          if (config.displayed_numeric_columns.includes(column)) {
            return filterByRange(cellValue, filters[column]);
          } else  {
              if (config.unique_column_values[column] <= 10) {
                return filterByCheckbox(cellValue, filters[column])
              }
              return filterByString(cellValue, filters[column])
              }
```
Would work as well i guess

Reply:
And also can you check if the multiple filter popup error still goes ?

Reply:
> It is to add filtering to the columns that are additional such as birth season but only
> 
> ```
>           if (config.displayed_numeric_columns.includes(column)) {
>             return filterByRange(cellValue, filters[column]);
>           } else  {
>               if (config.unique_column_values[column] <= 10) {
>                 return filterByCheckbox(cellValue, filters[column])
>               }
>               return filterByString(cellValue, filters[column])
>               }
> ```
> 
> Would work as well i guess

This would be even cleaner imo:

```js
if (config.displayed_numeric_columns.includes(column)) {
    return filterByRange(cellValue, filters[column]);
} else if (config.unique_column_values[column] <= 10) {
    return filterByCheckbox(cellValue, filters[column])
} else {
    return filterByString(cellValue, filters[column])
}
```

Reply:
> And also can you check if the multiple filter popup error still goes ?

Yes I just checked and theres only one pop up now perfect! The text `Filter by movie` can be omitted though. Only takes up space.